pkg/controller/controller_broker.go

```diff
@@ -221,7 +221,7 @@ func (c *controller) reconcileClusterServiceBrokerKey(key string) error {
 // error is returned to indicate that the binding has not been fully
 // processed and should be resubmitted at a later time.
 func (c *controller) reconcileClusterServiceBroker(broker *v1alpha1.ClusterServiceBroker) error {
-	glog.V(4).Infof("Processing ClusterServiceBroker %v", broker.Name)
+	glog.V(4).Infof("ClusterServiceBroker %q: reconciling", broker.Name)
```

Comment:
Let's keep "processing" so that the start of resource reconciliation events is still searchable, since every other resource has "Processing" here. We can change it to reconciling all at once with no harm done once the other resources' formats match.